.changeset/green-cherries-fail.md

```diff
@@ -0,0 +1,5 @@
+---
+'@hyperlane-xyz/registry': minor
```

Comment:
I think we established that removals should be major bumps

Reply:
nice catch, fat fingered it :D